src/xdocs/config_javadoc.xml

```diff
@@ -2181,24 +2293,126 @@ public class Test {
   &lt;property name="excludeScope" value="package"/&gt;
 &lt;/module&gt;
         </source>
+        <p>Example:</p>
+        <source>
+/**
+ * @author a
+ * @version $Revision1$
+ */
+public class ClassA { // OK
+    /** */
+    private class ClassB {} // OK
+}
+
+/**
+* @author
+* @version abc
+*/
+public class ClassC {} // OK
+
+/** */
+public class ClassD&lt;T&gt; {} // OK
+
+/** */
+private class ClassE&lt;T&gt; {} // violation, as param tag for &lt;T&gt; is missing
+
+/** */
+@Generated
+public class ClassF&lt;T&gt; {} // OK
+        </source>
+
+        <p>
+          To configure the check that allows missing <code>@param</code> tags:
+        </p>
+        <source>
+&lt;module name="JavadocType"&gt;
+  &lt;property name="allowMissingParamTags" value="true"/&gt;
+&lt;/module&gt;
+        </source>
+        <p>Example:</p>
+        <source>
+/**
+ * @author a
+ * @version $Revision1$
+ */
+public class ClassA { // OK
+    /** */
+    private class ClassB {} // OK
+}
+
+/**
+ * @author
+ * @version abc
+ */
+public class ClassC {} // OK
+
+/** */
+public class ClassD&lt;T&gt; {} // OK, as missing param tag is allowed
+
+/** */
+private class ClassE&lt;T&gt; {} // OK, as missing param tag is allowed
+
+/** */
+@Generated
+public class ClassF&lt;T&gt; {} // OK
+        </source>
 
         <p>
-          Example that allows missing comments for classes annotated with
-          <code>@SpringBootApplication</code> and <code>@Configuration</code>:
+            To configure the check that allows unknown tags:
         </p>
         <source>
-@SpringBootApplication // no violations about missing comment on class
-public class Application {}
+&lt;module name="JavadocType"&gt;
+  &lt;property name="allowUnknownTags" value="true"/&gt;
+&lt;/module&gt;
+        </source>
+        <p>Example:</p>
+        <source>
+/**
+ * @author a
+ * @version $Revision1$
+ */
+public class ClassA { // OK
+  /** */
+  private class ClassB {} // OK
+}
 
-@Configuration // no violations about missing comment on class
-class DatabaseConfiguration {}
+/**
+ * @unknownTag value // OK, as unknown tag is allowed
```

Comment:
please put this tag in all (except for last) other examples, to let users see violation on it by default.

Reply:
yeah sure. I have added it

Reply:
Github, generate web site